pkgs/by-name/fv/fvm/package.nix

```
@@ -0,0 +1,52 @@
+{
+  buildDartApplication,
+  clang,
+  pkg-config,
+  gtk3,
+  cmake,
+  fetchFromGitHub,
+  nix-update-script,
+  lib,
+}:
+
+buildDartApplication rec {
+  pname = "fvm";
+  version = "3.2.1";
+
+  src = fetchFromGitHub {
+    owner = "leoafarias";
+    repo = pname;
+    tag = version;
+    hash = "sha256-i7sJRBrS5qyW8uGlx+zg+wDxsxgmolTMcikHyOzv3Bs=";
+  };
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  nativeBuildInputs = [
+    clang
+    pkg-config
+    gtk3
+    cmake
+  ];
```

```suggestion
  nativeBuildInputs = [
    pkg-config
    cmake
  ];
  
  buildInputs = [ gtk3 ];
```

If you really need `clang` instead of `gcc`, please override stdenv for `buildDartApplication`:

```nix
buildDartApplication.override { stdenv = clangStdenv; } { ... }
```
```